### ISSUE TITLE
listfunds field removed in CLN breaks swaps

### ISSUE THREAD

**Issue by grubles** [ElementsProject/peerswap#182]
In CLN, the `channel_sat` field in `listfunds` is now removed. 

Peerswap relies on `channel_sat` by way of `fundingChannels.ChannelSatoshi` to check the swap amount against what this field provides:

```
	if fundingChannels.ChannelSatoshi < amountSat {
		return errors.New("not enough outbound capacity to perform swapOut")
```

Without it, we will need to use a different field in `listfunds` (`our_amount_msat` ?).

**Comment by nepet** [comment]
Some notes from my side:

The field that `ChannelSatoshi` is pointing to is `channel_total_sat` and was afaict deprecated in `v0.11.0`. The field represents, according to the documentation `channel_sat - available satoshis on our node’s end of the channel (values rounded down to satoshis as internal storage is in millisatoshi)`.

We need to replace this with `our_amount_msat`. Even better would be a replacement with `spendable_msat` which we can get from `lightning-listpeers`. It is a better estimate on what can be spend on a channel.

I also don't like the architecture here too much as it mixes presentation with data aggregation and business logic.

**Comment by grubles** [comment]
Discovered another field removal that affects Peerswap:

`listfunds` does not have an onchain outputs `value` field anymore, so we need to use `amount_msat` going forward.

**Comment by grubles** [comment]
Should be fixed in #183
